user-service/src/main/java/uk/gov/gchq/palisade/service/user/model/AuditableUserResponse.java

```diff
@@ -109,7 +111,7 @@ default AuditableUserResponse withAuditErrorMessage(AuditErrorMessage audit) {
 
 
             /**
-             * Compose value
+             * Adds the UserResponse to the {@link AuditableUserResponse}
```

Comment:
Add a basic reason as to why one migt be null 
For example: 'Adds a UserResponse and a AuditErrorMessage if there was a issue retrieving the user in the service"
ditto on line 124:126

Reply:
Re-worded the JavaDocs in this file to make it more obvious as to why `null` could be added.
Line 124 has also been updated.